WhaleSpotting/react-app/src/styles/ReportSighting.scss

```diff
@@ -80,4 +81,29 @@
         appearance: none;
         margin: 0; 
     }
+    
+    @media (max-width: 544px) {
+        .sighting-details .input-box{
+            margin-bottom: 15px;
+            width: 100%;
+            font-size: $mobileBodyFontSize;
+        }
+        .title{
+            font-size: $mobileTitleFontSize;
+        }
+        .card-component{
+            padding-right: 7px;
+        }
+        .submit-button{
+            margin: 10px 0 25px 0;
+        }
+
+    }
+    @media (max-width: $minMobileScreenSize){
```

Comment:
we should also add changing the margin bottom to mobile footerHeight in this

Reply:
Added as requested

Reply:
The change should edit the padding-bottom of the root report-sighting class, not the container. Also it should happen in the second media query not the first